### DISCUSSION TITLE
How to change external port wg-easy listens to?

### DISCUSSION THREAD

**Discussion by suxscribe** [wg-easy/wg-easy#731] [Q&A] [answered]
Hi!
I'd like to use wireguard on a custom port, smthn like **51829**

I've changed the port in the container installation command as follows:

```
docker run -d \
--name=wg-easy \
-e WG_HOST=194.35.116.234 \
-e PASSWORD=PASSWORD \
-v ~/.wg-easy:/etc/wireguard \
-p 51829:51820/udp \
-p 51821:51821/tcp \
--cap-add=NET_ADMIN \
--cap-add=SYS_MODULE \
--sysctl="net.ipv4.conf.all.src_valid_mark=1" \
--sysctl="net.ipv4.ip_forward=1" \
--restart unless-stopped \
ghcr.io/wg-easy/wg-easy
```

Container is up, web ui is working, but i can't connect to Wireguard. Is there a way to make it work?

**Comment by leesonaa** [comment, marked as answer]
-e WG_PORT=51829

https://github.com/wg-easy/wg-easy?tab=readme-ov-file#options

**Reply by suxscribe** [reply]
It worked, thanks!
